common/src/main/java/com/wynntils/features/stateless/FixPacketBugsFeature.java

```diff
@@ -22,7 +22,7 @@
 import net.minecraftforge.eventbus.api.EventPriority;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
-public class FixPacketBugsFeature extends InternalFeature {
```

Comment:
This should probably rather be a debug feature. Or a feature that the user can turn off, if it starts to cause crashes.

Reply:
Done